docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -1411,12 +1400,10 @@ to load, select and modify the analysis modules are documented here.
 
         ::
 
-                ANALYSIS_LOAD   A1  analysis.so
                 ANALYISIS_COPY  A1  A2
```

Comment:
Outside the diff of this PR, but perhaps fix this typo while you are here?

Reply:
Done